src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -75,6 +75,106 @@ inline int fastrand()
     return (g_seed>>16)&0x7FFF;
 }
 
+inline void rpp_host_rng_xorwow_f32_initialize_8seed_stream(RpptXorwowState *xorwowInitialState, Rpp32u seed)
```

Comment:
it is better to define it as template function with #seeds as template parameter for 4 seed and 8 seed cases

Reply:
Done